saleor/tax/migrations/0006_auto_20220830_1555.py

```diff
@@ -1,33 +0,0 @@
-# Generated by Django 3.2.14 on 2022-08-30 15:55
```

Comment:
This migration should have been removed in earlier PR but I missed it. These changes were moved to tax/0001_initial migration.